builds/uninstalling-openshift-builds.adoc

```diff
@@ -0,0 +1,23 @@
+:_content-type: ASSEMBLY
+[id="uninstalling-openshift-builds"]
+= Uninstalling {builds-shortname}
+include::_attributes/common-attributes.adoc[]
+:context: uninstalling-openshift-builds
+
+toc::[]
+
+Cluster administrators can uninstall the {builds-title} Operator by performing the following tasks:
+
+* Delete the Custom Resources (CRs) that were added by default when you installed the {builds-title} Operator.
+* Uninstall the {builds-title} Operator.
+
+Uninstalling only the Operator will not remove the {builds-title} components created by default when the Operator is installed.
```

Comment:
```suggestion
* Uninstall the {builds-title} Operator.
+
NOTE: Uninstalling only the Operator does not remove the {builds-title} components that were created by default when you installed the Operator.
```

Reply:
I would suggest reverse the sequence of tasks (unless there's a technical mandate to follow this sequence which I believe there isn't) which would be:

1. Uninstall Operator
2. Delete CRs. 
    Note: Uninstalling the Operator does not automatically remove the CRs and the related components.

Reply:
@snarayan-redhat I got it confirmed from SME and we cannot reverse the sequence. I have changed the bullet points to numbered sequence because the task of deleting CRs must be done before uninstalling the Operator.